service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -143,6 +159,48 @@ public void notifyLocalAuthorityOfIssuedStandardDirectionsOrder(StandardDirectio
         sendNotification(STANDARD_DIRECTION_ORDER_ISSUED_TEMPLATE, email, parameters, reference);
     }
 
+    @EventListener
+    public void sendNotificationsForIssuedCaseManagementOrder(CMOEvent event) {
+        EventData eventData = new EventData(event);
+
+        sendCMOCaseLinkNotifications(eventData.getCaseDetails(), eventData.getLocalAuthorityCode());
+    }
+
+    private void sendCMOCaseLinkNotificationForLocalAuthority(final CaseDetails caseDetails,
+                                                              final String localAuthorityCode) {
+        final String localAuthorityName = localAuthorityNameLookupConfiguration.getLocalAuthorityName(
+            localAuthorityCode);
+
+        Map<String, Object> localAuthorityNotificationParameters = caseManagementOrderEmailContentProvider
+            .buildCMOIssuedCaseLinkNotificationParameters(caseDetails, localAuthorityName);
+
+        String email = inboxLookupService.getNotificationRecipientEmail(caseDetails, localAuthorityCode);
+        sendNotification(CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE, email,
+            localAuthorityNotificationParameters, Long.toString(caseDetails.getId()));
+    }
+
+    private void sendCMOCaseLinkNotifications(final CaseDetails caseDetails, final String localAuthorityCode) {
```

Comment:
Not the most important but could this come before the method above, bit easier to follow the flow then